### PR TITLE
Assign keys to accounts

### PR DESCRIPTION
**Pull request by T-Dnzt**
Issue/Task Number: #836 
#836 

# Overview

This PR adds two new endpoints to assign keys to accounts as well as creating a key associated with an account on creation.

# Changes

- Add `/account.assign_key`
- Add `/account.unassign_key`
- Update `/key.create` to accept `account_id` and `role_name`

### CODE REVIEW

File: apps/admin_api/priv/swagger/account/request_bodies.yaml [hunk 1]
```diff
@@ -198,6 +198,47 @@ AccountUnassignUserBody:
...
+          type: string
+        role_name:
+          type: string
+        redirect_url:
+          type: string
+      required:
+        - key_id
+        - account_id
+        - role_name
+        - redirect_url
```
Comment (unnawut) [inline apps/admin_api/priv/swagger/account/request_bodies.yaml]: Key assignment needs `redirect_url`?

File: apps/ewallet/lib/ewallet/permissions/bouncer/targets/key_target.ex [hunk 2]
```diff
@@ -25,11 +25,11 @@ defmodule EWallet.Bouncer.KeyTarget do
   end
 
   @spec get_target_types() :: [:keys]
-  def get_target_types, do: [:keys]
+  def get_target_types, do: [:access_keys]
```
Comment (unnawut) [inline apps/ewallet/lib/ewallet/permissions/bouncer/targets/key_target.ex]: Also `@spec ... [:keys]` above

File: apps/ewallet/lib/ewallet/permissions/bouncer/targets/key_target.ex [hunk 3]
```diff
@@ -25,11 +25,11 @@ defmodule EWallet.Bouncer.KeyTarget do
...
 
-  def get_target_type(_), do: :keys
+  def get_target_type(_), do: :access_keys
 
-  @spec get_target_type(Key.t()) :: :keys
+  @spec get_target_type(Key.t()) :: [Account.t()]
```
Comment (unnawut) [inline apps/ewallet/lib/ewallet/permissions/bouncer/targets/key_target.ex]:
```suggestion
  @spec get_target_accounts(Key.t()) :: [Account.t()]
```

**Comment by T-Dnzt** [comment]
- [ ] Missing `global_role` in key serializer
- [ ] Remove `parent_id` from account.create params
- [ ] Prevent key creation if `account_id` is specified without a `role_name`

**Pull request merged**